pkg/pipelines/clustertasks.go

```diff
@@ -0,0 +1,29 @@
+package pipelines
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/openshift-pipelines/release-tests/pkg/assert"
+	"github.com/openshift-pipelines/release-tests/pkg/clients"
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func GetClusterTask(c *clients.Clients, clusterTaskName, status string) {
```

Comment:
I would expect that a this function returns a cluster task but it actually asserts its status. I suggest to renaming it to `AssertClusterTaskStatus`